front-end/src/app/layout/sidebar/menus/f3x/f3x-menu.component.ts

```diff
@@ -56,7 +58,13 @@ export class F3XMenuComponent extends AbstractMenuComponent {
         visible: isEditable,
       },
     ];
+    const editReportIsVisible =
```

Comment:
since the decision is being made here, does it make sense to just add or remove the item rather than always adding it and flipping visibility inside it?

Reply:
nice catch.  changed to dynamically adding item.